playbooks/common/openshift-node/config.yml

```diff
@@ -46,6 +47,8 @@
                                                     | union(groups['oo_etcd_to_config'] | default([])))
                                                 | oo_collect('openshift.common.hostname') | default([]) | join (',')
                                                 }}"
+  pre_tasks:
+  - include: pre_install_node.yml
   roles:
```

Comment:
If I use
```yaml
- include_role:
    name: openshift_node
  vars:
    openshift_ca_host: "{{ groups.oo_first_master.0 }}"
```
instead, the `restart NetworkManager` and `restart dnsmasq` handlers get skipped:

```
RUNNING HANDLER [openshift_node_dnsmasq : restart NetworkManager] **************
skipping: [10.8.170.180] => {
    "changed": false, 
    "skip_reason": "Conditional result was False", 
    "skipped": true
}

RUNNING HANDLER [openshift_node_dnsmasq : restart dnsmasq] *********************
skipping: [10.8.170.180] => {
    "changed": false, 
    "skip_reason": "Conditional result was False", 
    "skipped": true
}
META: ran handlers
```
even though there is no `when: ` condition attached to any of the handlers.